app/api/domains/api.py

```diff
@@ -38,6 +39,7 @@
 # or keep up with changes to https://github.com/JKBGL/gulag-api-docs.
 
 # Unauthorized (no api key required)
+# GET /search: returns a list of matching users, based on a passed string, sorted by ascending ID.
```

Comment:
ideally, this should be something like `GET /players`, but since the whole API is currently wrong, let's keep it consistently wrong 🥲 
```suggestion
# GET /get_players: returns a list of matching users, based on a passed string, sorted by ascending ID.
```

Reply:
this puts me thru great pain